bridge/qb/client/functions.lua

```diff
@@ -192,11 +192,11 @@ functions.IsWearingGloves = qbx.isWearingGloves
 ---@deprecated use lib.getClosestPlayer from ox_lib
 functions.GetClosestPlayer = function(coords) -- luacheck: ignore
     coords = type(coords) == 'table' and vec3(coords.x, coords.y, coords.z) or coords or GetEntityCoords(cache.ped)
-    local players = GetActivePlayers()
+    local players = lib.getNearbyPlayers(coords, 5, false)
```

Comment:
Use https://overextended.dev/ox_lib/Modules/GetClosestPlayer/Shared#libgetclosestplayer

Reply:
Right, now that I realized qb-core also has a max distance.